WiseBuy/Managers/DatabaseManager.m

```diff
@@ -46,8 +63,8 @@ +(void)registerUser:(User *)user withCompletion:(void(^)(BOOL success, NSError *
 + (void)logoutUser:(UIViewController *)vc {
     SceneDelegate *sceneDelegate = (SceneDelegate *) vc.view.window.windowScene.delegate;
     UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
-    UINavigationController *initialNavigationController = [storyboard instantiateViewControllerWithIdentifier:@"InitialNavigationController"];
-    sceneDelegate.window.rootViewController = initialNavigationController;
+    UIViewController *initialViewController = [storyboard instantiateViewControllerWithIdentifier:kLoginViewController];
```

Comment:
Why did this change from UINavigationController to UIViewController?

Reply:
Previously navigating to the UINavigationController disturbed the layout of the LoginViewController and Sign In button didn't work. So I change it to be the ViewController itself and now it works.

Reply:
Makes sense - when you have unrelated changes like that, it's good to either (1) separate it out into a separate PR or (2) explicitly call it out in your PR description. Otherwise the reviewer may think it's an accidental change or be unaware of the reasoning behind the change

Reply:
fixed